examples/webpack.config.js

```diff
@@ -230,6 +230,18 @@ module.exports = env => {
         resolve('public/'),
         resolve('dist/'),
       ],
+
+      proxy: {
```

Comment:
Unrelated change?

Reply:
Yes, this was necessary in order to be able to test performance on mobile. It's cherry-picked into a separate PR (https://github.com/cognitedata/reveal/pull/391)